lnrpc/signrpc/signer_server.go

```diff
@@ -547,3 +567,34 @@ func (s *Server) DeriveSharedKey(_ context.Context, in *SharedKeyRequest) (
 
 	return &SharedKeyResponse{SharedKey: sharedKeyHash[:]}, nil
 }
+
+// parseRawKeyBytes checks that the provided raw public key is valid and returns
+// the public key.
```

Comment:
The godoc comment should describe the behavior that a nil public key is returned if the length of the bytes is zero.

Reply:
added